package.json

```
@@ -45,11 +45,10 @@
   },
   "browserslist": "Node 10.17 - 11 and Node < 11, Node 12 - 13 and Node < 13, Node >= 13.7, > 0.5%, not OperaMini all, not dead",
   "peerDependencies": {
-    "graphql": "14 - 15",
-    "subscriptions-transport-ws": "^0.9.0"
+    "graphql": "^14.0.0 || ^15.0.0"
```

Can we please revert this syntax change?

```suggestion
    "graphql": "14 - 15"
```